esphome/components/x9c/output.py

```diff
@@ -26,6 +27,7 @@
             cv.Optional(CONF_INITIAL_VALUE, default=1.0): cv.float_range(
                 min=0.01, max=1.0
             ),
+            cv.Optional(CONF_STEP_DELAY, default=1): cv.int_range(min=1, max=100),
```

Comment:
This should be validated as a time period correctly
```suggestion
            cv.Optional(CONF_STEP_DELAY, default=1): cv.All(
                cv.positive_time_period_microseconds, 
                cv.Range(
                    min=cv.TimePeriod(microseconds=1), 
                    max=cv.TimePeriod(microseconds=100),
                )
            ),
```

Reply:
Should I create a new PR as this is already closed?